src/node/server/serverPluginModuleRewrite.ts

```diff
@@ -367,14 +367,16 @@ function isOptimizedCjs(root: string, importer: string, id: string) {
   return !!analysis.isCommonjs[slash(path.relative(root, modulePath))]
 }
 
+type ImportName = { importedName: string; localName: string }
+
```

Comment:
Maybe it is better with `ImportSpecifier`, what do you think ?

Reply:
> Maybe it is better with `ImportSpecifier`, what do you think ?

make sence